server/gitlab/api.go

```diff
@@ -681,8 +681,8 @@ func (g *gitlab) GetYourProjects(ctx context.Context, user *UserInfo) ([]*intern
 	return projects, nil
 }
 
-func (g *gitlab) GetLabels(ctx context.Context, user *UserInfo, projectID string) ([]*internGitlab.Label, error) {
-	client, err := g.gitlabConnect(*user.Token)
+func (g *gitlab) GetLabels(ctx context.Context, user *UserInfo, projectID string, token *oauth2.Token) ([]*internGitlab.Label, error) {
+	client, err := g.gitlabConnect(*token)
```

Comment:
Do we need to use `useGitlabClient` here?

Reply:
@mickmister We can use it but the main use of `useGitlabClient` is to fetch the oauth token, but here we already have the token in the params. Should we still use it here?

Reply:
Oh right, this `gitlab` struct doesn't use `useGitlabClient` function